apps/web/src/pages/Departments/UpdateDepartmentPage.tsx

```diff
@@ -47,42 +77,66 @@ export const DepartmentForm_Fragment = graphql(/* GraphQL */ `
       en
       fr
     }
+    orgIdentifier
+    size {
+      value
+      label {
+        localized
+      }
+    }
+    isCorePublicAdministration
+    isCentralAgency
+    isScience
+    isRegulatory
   }
 `);
 
-type FormValues = UpdateDepartmentInput;
-
 interface UpdateDepartmentProps {
   query: FragmentType<typeof DepartmentForm_Fragment>;
+  optionsQuery: FragmentType<typeof DepartmentFormOptions_Fragment>;
   handleUpdateDepartment: (
     id: string,
-    data: FormValues,
+    data: UpdateDepartmentInput,
   ) => Promise<FragmentType<typeof DepartmentForm_Fragment>>;
 }
 
 export const UpdateDepartmentForm = ({
   query,
+  optionsQuery,
   handleUpdateDepartment,
 }: UpdateDepartmentProps) => {
   const intl = useIntl();
   const navigate = useNavigate();
   const paths = useRoutes();
-  const initialDepartment = getFragment(DepartmentForm_Fragment, query);
+  const department = getFragment(DepartmentForm_Fragment, query);
   const methods = useForm<FormValues>({
     defaultValues: {
-      departmentNumber: initialDepartment.departmentNumber,
-      name: initialDepartment.name,
+      departmentNumber: department.departmentNumber,
+      name: department.name,
+      orgIdentifier: department.orgIdentifier,
+      size: department.size?.value,
+      departmentType: [
```

Comment:
Huh, I had to do a double-take on this, but it's cool.  Nice use of `satisfies`.

Reply:
Yeah, I was getting type errors because it did not recognize the strings as satisfying the type and then I remembered that keyword :sweat_smile: